js/index.js

```diff
@@ -1 +1,38 @@
 // Your code goes here
+
+window.addEventListener('load', (event) => alert('NOT A SCAM! You just won a free trip to a beach somewhere!'));
+
+let aNav = document.querySelectorAll('.nav-link');
```

Comment:
spoke with you on a possible fix for this, this error trickles down to the rest so my advice should be able to get you through all of the rest once you solve this one.